behave/features/steps/specialist_provider.py

```diff
@@ -415,15 +422,17 @@ def step_impl(context):
     error_header = "//h2[text()='Please correct the following errors:']"
 
     context.csv_page = context.helperfunc.find_by_xpath("//*[@id='wrapper']")
+
```

Comment:
could do this or could try this? https://stackoverflow.com/a/36166132

Reply:
Cool, so if it's plural, e.g. find_elements <<< (s) it creates a list and does not throw exceptions as it's just an empty list and not an empty element. That is so much better for finding elements that are not in the DOM. 

I've refactored and remove the try catch elements now.